src/grimp/application/usecases.py

```diff
@@ -21,7 +22,7 @@ class NotSupplied:
 
 
 # This is an arbitrary number, but setting it too low slows down our functional tests considerably.
-MIN_NUMBER_OF_MODULES_TO_SCAN_USING_MULTIPROCESSING = 50
```

Comment:
`MULTIPLE_PROCESSES` instead of `MULTIPROCESSING` to avoid confusion with `multiprocessing` module